agimus_controller/ocps/ocp_croco_hpp.py

```diff
@@ -37,11 +36,8 @@ def __init__(
         # Data of the model
         self._rdata = self._rmodel.createData()
 
-        # Obtaining the last joint name and joint id
-        self._last_joint_name, self._last_joint_id, self._last_joint_frame_id = (
-            get_last_joint(self._rmodel)
-        )
-
+        # Obtaining the gripper frame id
+        self._effector_frame_id = self._rmodel.getFrameId("panda_hand_tcp")
```

Comment:
The name "panda_hand_tcp" should be a parameter at some point...

Reply:
you're right, that would be for another PR, by reading the same yaml we're currently using with the ros node I guess